golang/godep.md

```diff
@@ -110,6 +112,55 @@ godep: no packages can be updated
 $ godep update github.com/Clever/worker/...
 ```
 
+## How do I do X?
```

Comment:
nit: maybe change this to "How do I ..." and change "adding" -> "add" and "updating" -> "update" below